data/faq/application_template.md

```diff
@@ -0,0 +1,122 @@
+coala 2017 Application Template
+===============================
+
+```
+Student Info
+------------
+
+- Name:
+- GitHub username:
+- Alternative-/Nickname:
+- Email:
+- Telephone(mobile or home):
+- What city and country will you reside in during the summer?
```

Comment:
'city' can be millions of people, or 25,000 people.
When it is 25,000 people, it is creepy to tell people you dont know and trust.

I'd prefer this to be 'Which country?' followed by something like 'Which city (or other geographical boundary you feel comfortable sharing)?'